inference-engine/src/inference_engine/blob_transform.cpp

```diff
@@ -139,6 +140,12 @@ static inline void blob_copy_4d(Blob::Ptr src, Blob::Ptr dst) {
     case Precision::I8:
         blob_copy_4d_t<Precision::U8>(src, dst);
         break;
+    case Precision::U64:
+        blob_copy_4d_t<Precision::U64>(src, dst);
+        break;
+    case Precision::I64:
```

Comment:
do we need different specialization for 64 bits data type?

Reply:
BTW, do we need such specializations at all? Looks like nobody need them before.

Reply:
Ok will remove